app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/maps/MapsFragmentTest.kt

```diff
@@ -70,51 +71,44 @@ class MapsFragmentTest {
 
     @Test
     fun a_permissionAskPopUp() {
-        onData(anything())
-            .inAdapterView(withId(R.id.list_view_routes)).atPosition(0).perform(
-                click()
-            )
+        onView(withText(startsWith(FIRST_ROUTE_NAME_PREFIX))).perform(click())
         onView(withId(R.id.button_next)).perform(click())
         if (LocationUtils.hasLocationPopUp()) {
             assertTrue(device.findObject(UiSelector().textContains(buttonTextAllow)).exists())
         }
     }
 
-    private fun b_permissionAllowShowLocation() {
-        onData(anything())
-            .inAdapterView(withId(R.id.list_view_routes)).atPosition(0).perform(
-                click()
-            )
+    /*
+    @Test
+    fun b_permissionAllowShowLocation() {
+        onView(withText(startsWith(FIRST_ROUTE_NAME_PREFIX))).perform(click())
         onView(withId(R.id.button_next)).perform(click())
         if (LocationUtils.hasLocationPopUp()) {
             device.findObject(UiSelector().textContains(buttonTextAllow)).click()
         }
         assertTrue(checkLocationEnabled(testRule))
-    }
+    } */
+
```

Comment:
Should be decommented and failing is the correct behaviour as the location mechanism as been disabled unintentionally at some point. I am looking into it to bring it back